packages/pwa-kit-create-app/assets/pwa/pwa-kit.config.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+exports.template = () => `{
```

Comment:
Did you consider declaring this as a JavaScript object and string-ifying it later?

This would allow us validate that it is validate JSON, format it etc.

Reply:
👍